src/components/SQFormScrollableCardsMenuWrapper/SQFormScrollableCardsMenuWrapper.tsx

```diff
@@ -48,7 +50,23 @@ function getSelectedComponent(selectedTab, children) {
   return children;
 }
 
-export default function SQFormScrollableCardsMenuWrapper({title, children}) {
+interface SQFormScrollableCardsMenuWrapperProps {
+  /** At least one instance of SQFormScrollableCard where each has
+   * prop `isHeaderDisabled` === true
+   * AND
+   * each has a unique string `value` prop and `label` prop so we
+   * have a menu label and can match the `value` to what's selected
+   * in the popover menu.
+   */
+  children: JSX.Element | JSX.Element[];
```

Comment:
making note here of the fact that Element _does_ meet this component's use case (each child must have `props.value`), so JSX.Element (or React.ReactElement) is appropriately narrow, otherwise `ReactNode` would be too inclusive and allow children of types like `string`